build.sbt

```diff
@@ -682,6 +682,7 @@ lazy val scioJmh: Project = Project(
   classDirectory in Jmh := (classDirectory in Test).value,
   dependencyClasspath in Jmh := (dependencyClasspath in Test).value,
   libraryDependencies ++= Seq(
+    "org.hamcrest" % "hamcrest-all" % hamcrestVersion,
```

Comment:
@nevillelyh is it used? or did I missed something?

Reply:
It's needed by `CombineFnTester`.